scripts/Run-Self-Hosted-Runner-Test.ps1

```diff
@@ -179,8 +179,16 @@ if (-not $WaitResult) {
     $DriveFreeSpaceGB = GetDriveFreeSpaceGB -DriveSpecification $Env:SystemDrive
     Write-Log "Current available disk space: $DriveFreeSpaceGB GB`n"
 
+    # $TestProcess refers to 'cmd.exe' which ends up running the real test application.
```

Comment:
where does cmd come into play here? aren't we running the tests with Powershell?